File: android/versioned-abis/expoview-abi39_0_0/src/main/java/abi39_0_0/expo/modules/calendar/CalendarPackage.java
package abi39_0_0.expo.modules.calendar;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi39_0_0.org.unimodules.core.BasePackage;
import abi39_0_0.org.unimodules.core.ExportedModule;

public class CalendarPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new CalendarModule(context));
  }
}

File: android/versioned-abis/expoview-abi48_0_0/src/main/java/abi48_0_0/expo/modules/calendar/CalendarPackage.kt
package abi48_0_0.expo.modules.calendar

import android.content.Context
import abi48_0_0.expo.modules.core.BasePackage

class CalendarPackage : BasePackage() {
  override fun createExportedModules(context: Context) = listOf(CalendarModule(context))
}
